src/sgame/botlib/bot_nav_edit.cpp

```diff
@@ -160,6 +219,14 @@ void BotDebugDrawMesh()
 		duDebugDrawNavMeshPortals( &dd, *cmd.nav->mesh );
 	}
 
+	//if ( cmd.showobstacles )
```

Comment:
I'm not sure if there is any point making this optional. I think it's fine as is.

Reply:
Unless there is a severe performance impact, I think it's fine to make it on by default. If we find a performance impact, we can add the check back.